Resources/doc/crud_interface.rst

```diff
@@ -337,12 +297,36 @@ By default, the quick_export action uses the ``pim_custom_entity.action.quick_ex
                     route:       pim_customentity_quickexport
                     job_profile: csv_reference_data_quick_export
 
-You can define another job profile if you want to export in another format. The job profile should be defined as a job fixtures as all Akeneo PIM backend tasks.
+You can define another job profile if you want to export in another format. The job profile should be defined as a job fixtures as all Akeneo PIM backend tasks. For example:
+
+.. code-block:: yaml
+
+    # Resources/fixtures/acme_fixture_set/jobs.yml
+    
+    csv_reference_data_quick_export:
+        connector: Akeneo Mass Edit Connector
+        alias: reference_data_quick_export
+        label: CSV deference data quick export
+        type: quick_export
+        configuration:
+            delimiter:  ;
+            enclosure:  '"'
+            withHeader: true
+            filePath:   /tmp/reference_data_quick_export.csv
+
+If your installation is already set up, you need to run the following command:
+
+.. code-block:: shell
+
+    php app/console akeeo:batch:create-job "Akeneo Mass Edit Connector" "reference_data_quick_export" "quick_export" "csv_reference_data_quick_export" '{    "delimiter": ";",    "enclosure": "\\",    "withHeader": true,    "filePath": "/tmp/reference_data_quick_export.csv"}'
```

Comment:
typo here ake**n**eo:batch

This command should be add in installation section